src/ipolatev.f90

```diff
@@ -70,307 +110,260 @@ SUBROUTINE ipolatev_grid(IP,IPOPT,grid_in,grid_out, &
+  !! - (13): LONGITUDE OF FIRST GRID POINT, OCTETS 51-54
+  !! - (14): RESOLUTION AND COMPONENT FLAGS, OCTET 55
+  !! - (15): LATITUDE OF LAST GRID POINT, OCTETS 56-59
+  !! - (16): LONGITUDE OF LAST GRID POINT, OCTETS 60-63
+  !! - (17): I-DIRECTION INCREMENT, OCTETS 64-67
+  !! - (18): NUMBER OF PARALLELS BETWEEN POLE AND EQUATOR, OCTETS 68-71
+  !! - (19): SCANNING MODE, OCTET 72
+  !!
+  !! Polar Stereographic Azimuthal:
+  !! - (8):  NUMBER OF POINTS ALONG X-AXIS, OCTETS 31-34
+  !! - (9):  NUMBER OF POINTS ALONG Y-AXIS, OCTETS 35-38
+  !! - (10): LATITUDE OF FIRST GRID POINT, OCTETS 39-42
+  !! - (11): LONGITUDE OF FIRST GRID POINT, OCTETS 43-46
+  !! - (12): RESOLUTION AND COMPONENT FLAGS, OCTET 47
+  !! - (13): TRUE LATITUDE, OCTETS 48-51
+  !! - (14): ORIENTATION LONGITUDE, OCTETS 52-55
+  !! - (15): X-DIRECTION GRID LENGTH, OCTETS 56-59
+  !! - (16): Y-DIRECTION GRID LENGTH, OCTETS 60-63
+  !! - (17): PROJECTION CENTER FLAG, OCTET 64
+  !! - (18): SCANNING MODE FLAGS, OCTET 65
+  !!
+  !! Rotated Equidistant Cyclindrical:
+  !! - (8):  NUMBER OF POINTS ALONG A PARALLEL, OCTS 31-34
+  !! - (9):  NUMBER OF POINTS ALONG A MERIDIAN, OCTS 35-38
+  !! - (10): BASIC ANGLE OF INITIAL PRODUCTION DOMAIN, OCTETS 39-42
+  !! - (11): SUBDIVISIONS OF BASIC ANGLE, OCTETS 43-46
+  !! - (12): LATITUDE OF FIRST GRID POINT, OCTETS 47-50
+  !! - (13): LONGITUDE OF FIRST GRID POINT, OCTETS 51-54
+  !! - (14): RESOLUTION AND COMPONENT FLAGS, OCTET 55
+  !! - (15): LATITUDE OF LAST GRID POINT, OCTETS 56-59
+  !! - (16): LONGITUDE OF LAST GRID POINT, OCTETS 60-63
+  !! - (17): I-DIRECTION INCREMENT, OCTETS 64-67
+  !! - (18): J-DIRECTION INCREMENT, OCTETS 68-71
+  !! - (19): SCANNING MODE, OCTET 72
+  !! - (20): LATITUDE OF SOUTHERN POLE OF PROJECTION, OCTETS 73-76
+  !! - (21): LONGITUDE OF SOUTHERN POLE OF PROJECTION, OCTETS 77-80
+  !! - (22): ANGLE OF ROTATION OF PROJECTION, OCTS 81-84
+  !!
+  !! @param[in] igdtleni Number of elements of the grid definition
+  !! template array for the input grid. Corresponds to the gfld%igdtlen
+  !! component of the ncep g2 library gridmod data structure.
+  !!
+  !! @param[in] igdtnumo Grid definition template number for the output grid.
+  !! Corresponds to the gfld%igdtnum component of the
+  !! ncep g2 library gridmod data structure.
+  !! See "igdtnumi" for specific template definitions.
+  !! Note: igdtnumo<0 means interpolate to random station points.
+  !!
+  !! @param[in] igdtmplo Grid definition template array for the output grid.
+  !! Corresponds to the gfld%igdtmpl component of the ncep g2 library gridmod data structure.
+  !! See "igdtmpli" for definition of array elements.
+  !!
+  !! @param[in] igdtleno Number of elements of the grid definition template array for the output grid.  c
+  !! Corresponds to the gfld%igdtlen component of the ncep g2 library gridmod data structure.
+  !!
+  !! @param[in] mi    Skip number between input grid fields if km>1 or dimension of input grid fields if km=1.
```

Comment:
Are the parameters all in the same order as the function declaration? Lots of the legacy documentation messed up the ordering to support INPUT vs. OUTPUT, but in the doxygen we want the same exact ordering as in the function declaration...

Reply:
Yes, I made sure they were in the same order.